trunk/ii02219/task_04/src/Untitled-1.py

```diff
@@ -0,0 +1,202 @@
+import pygame
+import sys
+import os
+
+# Инициализация Pygame
+pygame.init()
+game_over_flag = False
+
+
+# Установка размеров окна
+WIDTH, HEIGHT = 800, 400
+win = pygame.display.set_mode((WIDTH, HEIGHT))
+pygame.display.set_caption("Dino Game")
+
+# Загрузка изображения заднего фона
+background_image = pygame.image.load('C:/Users/dende/Desktop/task_04_05/clouds.png')

```

Comment:
```suggestion
background_image = pygame.image.load('../images/clouds.png')

```
Остальные пути также сделать + переименовать файл Untitled-1.py